packages/react/src/components/pagination/Pagination.test.tsx

```diff
@@ -132,4 +132,46 @@ describe('<Pagination /> spec', () => {
     const bothEllipsisHighCase = getAllByText(bothEllipsisHighCaseFragment().firstChild as HTMLElement, '...');
     expect(bothEllipsisHighCase.length).toEqual(2);
   });
+
+  it('should show screen reader notification when user changes page', () => {
+    const { rerender, queryByText } = render(
+      <Pagination
+        onChange={() => null}
+        paginationAriaLabel="Pagination"
+        pageCount={25}
+        pageIndex={3}
+        pageHref={() => '#'}
+        language="en"
+        siblingCount={1}
+      />,
+    );
+    expect(queryByText('Opened page 4')).not.toBeInTheDocument();
+
+    rerender(
```

Comment:
Would this work with a user click?

Reply:
The user click handling is handled by the user of the component. So the test would only work on specific case, and it would basically test the same thing, as changing the prop is the trigger for the notification.

Reply:
Yes true 👍